includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -59,14 +59,14 @@ class WC_Payment_Gateway_WCPay extends WC_Payment_Gateway_CC {
 	 *
 	 * @var WC_Payments_Customer_Service
 	 */
-	private $customer_service;
+	protected $customer_service;
 
 	/**
 	 * WC_Payments_Token instance for working with customer tokens
 	 *
 	 * @var WC_Payments_Token_Service
 	 */
-	private $token_service;
+	protected $token_service;
```

Comment:
Not sure why do we need to change the visibility here. Can we keep those fields private?

Reply:
This is necessary so that I can access it from `WC_Payment_Gateway_WCPay_Subscriptions_Compat`.